### ISSUE TITLE
Basic Auth prevents access to config.yml

### ISSUE THREAD

**Issue by cpsubrian** [netlify/netlify-cms#202]
**- Do you want to request a *feature* or report a *bug*?**
Feature

**- What is the current behavior?**
Request to /admin/config.yml fails (401) due to basic auth headers in the `_headers` file.

**- What is the expected behavior?**
Need a way for the config.yml request to work. Since the admin has its own authentication, is there a way to **exclude** paths that the basic auth picks up on?

**Comment by cassiozen** [comment]
Hi @cpsubrian - sorry but I didn't understand. Where are you serving the files from? Can you share your setup?

**Comment by cpsubrian** [comment]
I am serving the site from Netlify. My `docroot` directory contains:

**_headers**
```
/*
  Basic-Auth: username:password
```

**admin/index.html**
```
<!doctype html>
<html>
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />

  <title>Content Manager</title>
  <!-- Include the stylesheets from your site here -->
  <link rel="stylesheet" href="https://unpkg.com/netlify-cms@^0.1/dist/cms.css" />
  <!-- Include a CMS specific stylesheet here -->
</head>
<body>
  <script src="https://unpkg.com/netlify-cms@^0.1/dist/cms.js"></script>
</body>
</html>
```

**admin/config.yml**
```
backend:
  name: github
  repo: mygithub/repo
  branch: master
collections: # A list of collections the CMS should be able to edit
  - name: "page" # Used in routes, ie.: /admin/collections/:slug/edit
    label: "Page" # Used in the UI, ie.: "New Post"
    folder: "docs" # The path to the folder where the documents are stored
    create: true # Allow users to create new documents in this collection
    fields: # The fields each document in this collection have
      - {label: "Title", name: "title", widget: "string", tagname: "h1"}
      - {label: "Body", name: "body", widget: "markdown"}
```

When I hit `/admin` in the browser, the CMS starts to load but I see the following:

![screenshot 2017-01-05 11 22 44](https://cloud.githubusercontent.com/assets/319152/21694282/49b01074-d339-11e6-8425-536b6429355e.png)

I can confirm in Chrome Network debugger that the request to config.yml fails with a 401. It is missing the `authorization` header that would be required for it to be served.

**Comment by cassiozen** [comment]
@cpsubrian, We are working on a solution, but if you need to get this working urgently, there's a workaround:

1 - convert your configuration file format from YAML to JSON. (There are a few online services that does this)
2 - Embed the configuration in the index.html file, feeding it to the CMS through `window.CMS_CONFIG`. Example:

```javascript
<body>
  <script>
   window.CMS_CONFIG = {...} // Your config here
  </script>
  <script src="https://unpkg.com/netlify-cms@^0.2/dist/cms.js"></script>
```

**Comment by cassiozen** [comment]
Fixed on the last version (0.3.0). I will close for now, but please feel free to reopen if needed.
